{{ cookiecutter.safe_formal_name }}/app/build.gradle

```diff
@@ -1,24 +1,32 @@
 apply plugin: 'com.android.application'
+apply plugin: 'com.chaquo.python'
 
 android {
     compileSdkVersion 32
     defaultConfig {
         applicationId "{{ cookiecutter.package_name }}.{{ cookiecutter.module_name }}"
-
-        // JNI crashes may happen on older versions:
-        // https://github.com/beeware/rubicon-java/issues/74
-        minSdkVersion 26
-
-        targetSdkVersion 32
         versionCode {{ cookiecutter.version_code }}
         versionName "{{ cookiecutter.version }}"
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+
+        minSdkVersion 16
+        targetSdkVersion 32
+
+        python {
+            pip {
+                install "-r", "requirements.txt"
+            }
+        }
         externalNativeBuild {
             cmake {
                 cppFlags "-std=c++14"
             }
         }
+        ndk {
+            abiFilters "arm64-v8a", "armeabi-v7a", "x86_64"
```

Comment:
Actually, rather than making it a template variable, requiring a rerun of `briefcase create` and a full rebuild, maybe we can make `build.gradle` read it from a .properties file which is updated by `briefcase update`.

Reply:
+1 to making it templated in some fashion. 

Writing to a `.properties` file sounds like a viable approach for this specific problem in the Android case. The general problem of "update the app template after the app has been generated" is logged as beeware/briefcase#472, if we can fix the general problem, that would be an added bonus.